alluxio/worker_ring.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, host="localhost", port=2379, options=None):
                 "Both ETCD username and password must be set or both should be unset."
             )
 
-    def get_worker_addresses(self):
+    def get_worker_addresses(self) -> Set[WorkerNetAddress]:
```

Comment:
@ChunxuTang please modify the alluxio_file_system.py as well, the consumer as well and test out the code